Compilation failure on FreeBSD 13,2

```
opam install ocaml-lsp-server
[blah]
┌─ The following actions failed
│ λ build ocaml-lsp-server 1.15.1-5.0
└─
```
I believe the relevant error is ` submodules/lev/lev/vendor/ev.c:510:11: fatal error: sys/statfs.h: No such file or directory`. Indeed, there is no `sys/statfs.h` on FreeBSD; I think it is specific for GNU C library.

```
🌐 eldanna in ~
❯ uname -srm
FreeBSD 13.2-RELEASE amd64
🌐 eldanna in ~
❯ ocamlc --version
5.0.0
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,8 @@
 
 ## Fixes
 
+- Fix the build on FreeBSD, where `statfs(2)` is declared in `<sys/mount.h>`
+  rather than `<sys/statfs.h>`. (#2070, fixes #1069, @dayangac)
 - Keep construct-completion text edits on the request line when Merlin recovery
   returns a multiline location. (#2034, @rgrinberg)
 - Advertise Dune promotion code actions using their returned `quickfix` kind.

